fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
It'd be good to understand exactly what is in those fields before we try and set the interface in stone. Could you do that sooner rather than later?

We don't really have the types, since they're both `String` :-/

Design consistency absolutely is a goal, whether specific things matter is up to our product and design people though, not us (although the technical constraints (like us not supporting HTML) are obviously totally relevant here).